test-ci.js

```diff
@@ -185,38 +182,30 @@ let csvNA = null;
   };
 
   let testResult = async function() {
-    let backendModels = [
-      'Mac-MPS',
-      'Mac-BNNS',
-      'Mac-WASM',
-      'Mac-WebGL',
-      'Android-NNAPI',
-      'Android-WASM',
-      'Android-WebGL',
-      'Windows-clDNN',
-      'Windows-WASM',
-      'Windows-WebGL',
-      'Linux-clDNN',
-      'Linux-WASM',
-      'Linux-WebGL',
+    let backendModels =[
```

Comment:
Please use these keys
macOS-Polyfill-Fast-WASM
Android-Polyfill-Fast-WASM
Win-Polyfill-Fast-WASM
Linux-Polyfill-Fast-WASM
and also update baseline + config files
to align with RegressionChecker tools.